syncany-lib/src/main/java/org/syncany/operations/down/actions/FileCreatingFileSystemAction.java

```diff
@@ -70,20 +72,26 @@ protected void createFolder(FileVersion targetFileVersion) throws Exception {
 		}
 	}
 
-	protected void createFile(FileVersion reconstructedFileVersion) throws Exception {
-		File reconstructedFileInCache = assembleFileToCache(reconstructedFileVersion);		
-		moveFileToFinalLocation(reconstructedFileInCache, reconstructedFileVersion);	
+	protected void createFile(FileVersion reconstructedFileVersion) throws IOException {
+		File reconstructedFileInCache;
+		try {
+			reconstructedFileInCache = assembleFileToCache(reconstructedFileVersion);
```

Comment:
I've put the line in there, but normally I would not advocate this as the Exceptions we want to catch are from the `assembleFileToCatch` call and not the other one :wink: